### PR TITLE
Fixed a bug in inference and PyTorch UserWarnings while training

### PR DESCRIPTION
**Pull request by KapilM26**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (Flova) [inline .vscode/settings.json:1]: Please remove your IDE settings here

File: utils/datasets.py [hunk 2]
```diff
@@ -44,18 +44,13 @@ def __init__(self, folder_path, transform=None):
         self.transform = transform
 
     def __getitem__(self, index):
-
         img_path = self.files[index % len(self.files)]
```
Comment (Flova) [inline utils/datasets.py:47]: Why do you add the tensor conversion, padding, and resizing to the dataloader? It is done in the `self.transform` you removed. There is a file (`transforms.py`) which contains all these operations and when the dataloader is created a list containing the pipeline's transforms. These contain image padding, data augmentation, tensor conversion,... a minimal subset of transforms is provided with the name `DEFAULT_TRANSFORMS`

**Review by Flova — CHANGES_REQUESTED** [review]
First of all thank you for your contribution!

Could you provide further information which issues you try to fix?

I also left a few comments regarding the changes you made.
